### ISSUE TITLE
Add support for right-to-left languages

### ISSUE THREAD

**Issue by ZviRackover** [alextselegidis/easyappointments#217]
In order to translate to Arabic and Hebrew rtl support is required.

**Comment by alextselegidis** [comment]
Yeah I wanted to implemented this as well but I don't really know how the styles should be made. Does this only affect the text alignment or are there more to do than that?

**Comment by ZviRackover** [comment]
Text alignment would be a good start. Then changing the layout for RTL would be nice. For example, if for English a side menu would appear on the left, you would want it on the other side for RTL languages.

**Comment by alextselegidis** [comment]
Ok I see, I will implement this for sure in a future release. Thanks for posting :)
